filebeat/input/event.go

```diff
@@ -11,24 +11,37 @@ import (
 
 // Event is sent to the output and must contain all relevant information
 type Event struct {
-	common.EventMetadata
-	ReadTime     time.Time
-	InputType    string
-	DocumentType string
-	Bytes        int
-	Text         *string
-	JSONConfig   *reader.JSONConfig
-	State        file.State
-	Data         common.MapStr // Use in readers to add data to the event
+	EventMeta
+	Text       *string
+	JSONConfig *reader.JSONConfig
+	Data       common.MapStr // Use in readers to add data to the event
+
+}
+
+type EventMeta struct {
 	Pipeline     string
 	Fileset      string
 	Module       string
+	InputType    string
+	DocumentType string
+	common.EventMetadata
+	ReadTime time.Time
+	Bytes    int
+	State    file.State
+}
+
+type EventHolder struct {
```

Comment:
Can we rename `EventHolder` to `Data` to be consistent with https://github.com/elastic/beats/blob/master/libbeat/outputs/outputs.go#L20 ? I'm ok with keeping Metadata as here it is at the moment only about Metadata